etc/module.xml

```diff
@@ -24,7 +24,7 @@
 -->
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
 
-    <module name="Adyen_Payment" setup_version="4.5.4">
+    <module name="Adyen_Payment" setup_version="5.3.1">
```

Comment:
Is this the correct usage of setup_version according to our release convention? https://hub.is.adyen.com/our-company/organization/workstreams/mid-market-solution/plugins-and-partners/internal-repo/release-steps/magento2/#release-on-github

Reply:
Yes, I believe so.

Reply:
it's gonna be a feature release so I would go for ->

```suggestion
    <module name="Adyen_Payment" setup_version="5.4.0">
```